base/reflection.jl

```diff
@@ -588,7 +588,7 @@ isdispatchtuple(@nospecialize(t)) = (@_pure_meta; isa(t, DataType) && (t.flags &
 
 iskindtype(@nospecialize t) = (t === DataType || t === UnionAll || t === Union || t === typeof(Bottom))
 isconcretedispatch(@nospecialize t) = isconcretetype(t) && !iskindtype(t)
-using Core: has_free_typevars
+import Core: has_free_typevars
```

Comment:
Any particular reason for this change? `has_free_typevars` should not be overloadable anyways, right?

Reply:
ah, that's just because I'm almost always using `import` just because of https://github.com/JuliaLang/julia/issues/36529.

Reply:
Ok, makes sense